packages/pluggable-widgets-tools/test-config/jest.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     rootDir: join(projectDir, "src"),
     globals: {
         "ts-jest": {
+            isolatedModules: true,
```

Comment:
Why we have `ts-jest` options in `globals`?
We probably should put them in `transform` as [options for](https://kulshekhar.github.io/ts-jest/docs/getting-started/options#introduction) `ts-jest`

Reply:
Good point, no need at all for the ts-jest config to be available globally.

Fixed ✨